doc/sphinx/practical-tools/utilities.rst

```diff
@@ -123,23 +123,29 @@ the top directory of your package's source tree.  We recommend using the
 avoid putting your code into subdirectories such as "src" unless you want
 that to be part of the :term:`logical path` in Coq.
 
-For example, here's a minimal _CoqProject file for the `MyProject` package
-(the logical name of the package), which has two script files `File1.v`
-and `SubDir/File2.v` whose logical paths
-are `MyProject.File1` and `MyProject.SubDir.File2`.  `_CoqProject` should be
-in the top-level directory of your project.  We recommend naming that directory
-`MyProject`, the same as the logical name::
+For example, here's a minimal _CoqProject file for the `MyPackage` package
+(the logical name of the package), which includes all the ``.v`` files (and
+other file types) in the current directory and its subdirectories:
+
+  -R . MyPackage
+  .
```

Comment:
I think this little dot deserves a comment, the reader may not even see it. Could also be written `./` to make it more visible

Reply:
Anyway, I'd recommend putting the sources in a (sub) directory. I don't recall any project placing source files at the root.

Reply:
Ah, it was supposed to be formatted like this, which I fixed:

![image](https://user-images.githubusercontent.com/1253341/182046077-00947110-42d2-4e1f-806d-02bee3206789.png)

I'll refine the recommendations soon.  FWIW the bignums package puts files at the root.  Hard to expect people to know about or follow unwritten conventions.

```
~/.opam/__coq-platform.2022.01.0~8.15~beta1/.opam-switch/sources/coq-bignums.8.15.0$ more _CoqProject
-I plugin
-R . Bignums
```